reflex/components/markdown/markdown.py

```diff
@@ -199,7 +201,15 @@ def get_component(self, tag: str, **props) -> Component:
             raise ValueError(f"No markdown component found for tag: {tag}.")
 
         special_props = [_PROPS_IN_TAG]
-        children = [_CHILDREN]
+        children = [
+            _CHILDREN
+            if tag != "codeblock"
+            else ternary_operation(
+                ARRAY_ISARRAY.call(_CHILDREN),  # type: ignore
+                _CHILDREN.to(list)[0],
```

Comment:
shouldn't we concatenate the strings here instead of taking the first? (i'm not sure)